components/schemes/Color/src/MDCSemanticColorScheme.h

```diff
@@ -74,6 +74,18 @@
  @c backgroundColor.
  */
 @property(nonnull, readonly, copy, nonatomic) UIColor *onBackgroundColor;
+
+/**
+ A color used on the overlay when a surface is elevated.
```

Comment:
A color that is used for the elevation overlays, applied with an alpha on top of surfaces.

Reply:
Addressed in 0ee543f80c066811a44a4e8b54b788e22f2c0c4a